### PR TITLE
Homework 5

### PR DESCRIPTION
**Pull request by IsaevMH**

### CODE REVIEW

File: src/test/java/com/epam/tc/hw5/cucumber/exercise1/hook/CucumberHook.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
+package com.epam.tc.hw5.cucumber.exercise1.hook;
+
+import com.epam.tc.hw5.driver.Driver;
+import com.epam.tc.hw5.service.TestContext;
+import io.cucumber.java.After;
+import io.cucumber.java.Before;
+import org.openqa.selenium.WebDriver;
+
+public class CucumberHook {
```
Comment (ZeroShadows) [inline src/test/java/com/epam/tc/hw5/cucumber/exercise1/hook/CucumberHook.java:9]: you don't need 3 simular CucumberHook classes, remove excessive two

**Comment by ZeroShadows** [comment]
general comment: lets make a common CucumberHook, RunAcceptanceTest and Steps classes

File: src/test/java/com/epam/tc/hw5/cucumber/exercise1/hook/CucumberHook.java (hunk 1 above)
Comment (IsaevMH) [reply src/test/java/com/epam/tc/hw5/cucumber/exercise1/hook/CucumberHook.java:9]: Issue fixed

File: pom.xml [hunk 2]
```diff
@@ -69,11 +70,23 @@
...
+        <dependency>
+            <groupId>io.cucumber</groupId>
+            <artifactId>cucumber-java</artifactId>
+            <version>6.11.0</version>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/io.qameta.allure/allure-cucumber6-jvm -->
+        <dependency>
+            <groupId>io.qameta.allure</groupId>
+            <artifactId>allure-cucumber6-jvm</artifactId>
+            <version>2.14.0</version>
```
[the remark this answers is not in the document]
Comment (IsaevMH) [reply pom.xml:89]: Issue is fixed

**Comment by IsaevMH** [comment]
> 
> 
> general comment: lets make a common CucumberHook, RunAcceptanceTest and Steps classes



> 
> 
> general comment: lets make a common CucumberHook, RunAcceptanceTest and Steps classes
Issue is fixed

**Review by ZeroShadows — APPROVED** [review]

**Pull request merged**